examples/NSIDC/ISat2_from_CMR_to_S3.ipynb

```diff
@@ -0,0 +1,665 @@
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 17,
+   "id": "soviet-shannon",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# READ only temporary credentials\n",
+    "import s3fs\n",
+    "import h5py\n",
+    "\n",
+    "s3_cred = CMR_auth.get_s3_credentials()\n",
+    "\n",
+    "s3_fs = s3fs.S3FileSystem(key=s3_cred['accessKeyId'],\n",
+    "                          secret=s3_cred['secretAccessKey'],\n",
+    "                          token=s3_cred['sessionToken'])\n",
+    "\n",
+    "# Now you could grab S3 links to your cloud instance (EC2, Hub etc) using:\n",
+    "# s3_fs.get('s3://SOME_LOCATION/ATL03_20181015124359_02580106_004_01.h5', 'test.h5')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "1a29b8c4-f876-4d8f-8fc2-3e00fc83cafb",
+   "metadata": {},
+   "source": [
+    "### We now have the propper credentials and file mapper to access the data within AWS us-west-2. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "other-nowhere",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "with s3_fs.open('s3://nsidc-cumulus-prod-protected/ATLAS/ATL03/004/2018/10/15/ATL03_20181015124359_02580106_004_01.h5', 'rb') as s3f:\n",
+    "    with h5py.File(s3f, 'r') as f:\n",
+    "        print([key for key in f.keys()])"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "imposed-valuation",
+   "metadata": {},
+   "source": [
+    "### Using xarray to open files on S3\n",
+    "\n",
+    "ATL data is complex so xarray doesn't know how to extract the important bits out of it."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "laden-combine",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import xarray\n",
+    "\n",
+    "with s3_fs.open('s3://nsidc-cumulus-prod-protected/ATLAS/ATL03/004/2018/10/15/ATL03_20181015124359_02580106_004_01.h5', 'rb') as s3f:\n",
+    "    ds= xarray.open_dataset(s3f)\n",
+    "    for varname in ds:\n",
+    "        print(varname)\n",
+    "ds"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "noble-sense",
+   "metadata": {},
+   "source": [
+    "### \"Downloading\" files on S3 using the official aws-cli library\n",
+    "The quotes on downloading are because ideally you'll be working on an EC2 (virtual machine for short) instance on the us-west-2 region."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "enormous-accountability",
+   "metadata": {},
+   "outputs": [],
+   "source": []
+  }
+ ],
+ "metadata": {
+  "kernelspec": {
+   "display_name": "Python 3 (ipykernel)",
+   "language": "python",
+   "name": "python3"
+  },
+  "language_info": {
+   "codemirror_mode": {
+    "name": "ipython",
+    "version": 3
+   },
+   "file_extension": ".py",
+   "mimetype": "text/x-python",
+   "name": "python",
+   "nbconvert_exporter": "python",
+   "pygments_lexer": "ipython3",
+   "version": "3.8.10"
+  }
+ },
+ "nbformat": 4,
+ "nbformat_minor": 5
+}
```

Comment:
So you open the data ... and then you list the variable names. This isn't terribly interesting. I want a plot here. I want to see data on map.